rules/integrations/aws/privilege_escalation_sts_assumerole_abuse.toml

```diff
@@ -0,0 +1,60 @@
+[metadata]
+creation_date = "2021/05/17"
+maturity = "production"
+updated_date = "2021/07/24"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies the suspicious use of AssumeRole. Attackers could move laterally and escalate privileges.
+"""
+false_positives = [
+    """
+    Automated processes that uses Terraform may lead to false positives.
+    """,
+]
+index = ["filebeat-*", "logs-aws*"]
+language = "kuery"
+name = "AWS STS AssumeRole Abuse"
```

Comment:
Is this behavior different from the normal AssumeRole usage? If not, we need replace `Abuse` for `Usage`